app/views/course/user_invitations/index.html.slim

```diff
@@ -0,0 +1,23 @@
+- add_breadcrumb :index
+= page_header
+
+= render partial: 'course/users/tabs'
+
+- accepted_invitations = @invitations.each.select(&:confirmed?)
+- pending_invitations = @invitations.select { |invitation| !invitation.confirmed? }
```

Comment:
👍 You reminded me that there's `partition`.  I used `select` before cause it has better performance than array subtraction ( O(n) vs (O(m*n) )